assignments/objects.js

```diff
@@ -19,26 +19,71 @@ const example = {
 
 // Write your intern objects here:
 
+const intern1 = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy@psu.edu",
+  "gender": "F"
+}
+const intern2 = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
+}
+const intern3 = {
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+}
+const intern4 = {
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+}
+const intern5 = {
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F"
+}
+
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
+console.log(intern1["name"]);
 
 // Kennan's ID
+console.log(intern2["id"]);
 
 // Keven's email
+console.log(intern3["email"]);
 
 // Gannie's name
+console.log(intern4["name"]);
 
 // Antonietta's Gender
+console.log(intern5["gender"]);
 
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
+intern2.speak = function(){
+  console.log("Hello, my name is " + this.name);
+};
+intern2.speak();
+
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
 //console.log(antonietta.multiplyNums(3,4));
+intern5.multiplyNums = function(a,b){
+  console.log(a * b);
+}
+intern5.multiplyNums(3,4);
```

Comment:
Keep in mind the instructions here: you were asked to `return` the product and then `console.log` the function call. It is fair to cut out the middleman here (the `return`), though.